server/configuration.go

```diff
@@ -177,6 +204,13 @@ func (c *configuration) Clone() *configuration {
 		}
 	}
 
+	if c.ICEServersConfigs != nil {
+		cfg.ICEServersConfigs = make([]rtc.ICEServerConfig, len(c.ICEServersConfigs))
+		for i, u := range c.ICEServersConfigs {
+			cfg.ICEServersConfigs[i] = u
+		}
```

Comment:
nit: Could we do this?
```suggestion
		copy(cfg.ICEServersConfigs, c.ICEServersConfigs)
```
Same for 210